sqlitecluster/SQLiteNode.h

```diff
@@ -106,7 +107,9 @@ class SQLiteNode : public STCPNode {
     // This will broadcast a message to all peers, or a specific peer.
     void broadcast(const SData& message, Peer* peer = nullptr);
 
-    void setCommandAddress(const string& commandAddress);
+    // Takes two string in the form of `host:port` (i.e., `www.expensify.com:80` or `127.0.0.1:443`) and creates a
+    // similar string with the host from hostPart and the port from portPart .
```

Comment:
could you also leave a _why_ here, looking at where it's used I think this is to ensure the port is set to the `commandPort` passed here https://github.com/Expensify/Bedrock/blob/2e115c49fea887f70ed831cb1e5c201d576a86d9/sqlitecluster/SQLiteNode.cpp#L115 which is the new `_commandPortPrivate`?